fineract-provider/src/main/java/org/apache/fineract/organisation/teller/service/TellerManagementReadPlatformServiceImpl.java

```diff
@@ -762,8 +752,8 @@ public String cashierTxnSummarySchema() {
             sqlBuilder.append(" or acnttrans.to_savings_transaction_id = sav_txn.id) ");
             sqlBuilder.append("    where sav_txn.is_reversed = false and c.id = ? ");
             sqlBuilder.append(" and sav.currency_code = ? ");
-            sqlBuilder.append("    and o.hierarchy like ? ");
-            sqlBuilder.append("    and sav_txn.transaction_date between c.start_date and date_add(c.end_date, interval 1 day) ");
+            //sqlBuilder.append("    and o.hierarchy like ? ");
```

Comment:
comment.

Reply:
We are filtering transactions for a specific cashier and each cashier is already tagged to an office through the staff_id foreign key (that has the office_id foreign) so including it here seems redundant.